projects/auth0-angular/src/lib/auth.service.spec.ts

```diff
@@ -113,6 +113,25 @@ describe('AuthService', () => {
         });
       });
     });
+
+    it('should return true when retrieving a token succesful', (done) => {
```

Comment:
The test name reads a bit odd to me, I would suggest:

```suggestion
    it('should return true after successfully getting a new token', (done) => {
```